index.html

```diff
@@ -10,12 +10,67 @@
     <title>Apple</title>
   </head>
   <body>
-  <nav></nav>
     <main>
+      <div class ='main-img'>
+        <nav>
+          <div class='menu-div'>
+<ul class='menu-bar'>
+  <li class='menu-bar-li'><div><i class="fab fa-apple"></i></div></li>
+  <li class='menu-bar-li'><a>Store</a></li>
+  <li class='menu-bar-li'><a>Mac</a></li>
+  <li class='menu-bar-li'><a>iPod</a></li>
+  <li class='menu-bar-li'><a>iPhone</a></li>
+  <li class='menu-bar-li'><a>iPad</a></li>
+  <li class='menu-bar-li'><a>iTunes</a></li>
+  <li class='menu-bar-li'><a>Support</a></li>
+  <li class='search-menu'><div class='search-div'><i class="fas fa-search"></i><input /></div></li>
+
+</ul>
+</div>
+ </nav>
+
+ <div class='central-text'>
+   <div class='image'>
+      <img src='./img/first_img.png' alt=''/>
+   </div>
+   <div class='image'>
+     <img  src='./img/second_img.png' alt=''/>
+    </div>
+   <div class='image-div'>
+     <a href="#" class='image-a'>
+        <img src='./img/third_img.png' alt=''/>
+     </a>
       
+   </div>
+   
+  </div>
+</div>
+<div class='iphone-models-div'>
```

Comment:
This div should be move into the footer, which will be the top part of the footer.